### PR TITLE
Make pull-kubernetes-kubemark-e2e-gce-big manually triggered, optional

### PR DESCRIPTION
**Pull request by spiffxp**
part of https://github.com/kubernetes/test-infra/issues/18729

ref: https://github.com/kubernetes/test-infra/issues/18729#issuecomment-671252325

/cc @wojtek-t @mm4tt

### CODE REVIEW

**Comment by spiffxp** [comment]
pull-test-infra-gubernator failed!  the system works! (closing https://github.com/kubernetes/test-infra/issues/18770)

**Comment by spiffxp** [comment]
FYI @BenTheElder @liggitt

File: gubernator/config.yaml [hunk 1]
```diff
@@ -34,7 +34,6 @@ jobs:
   - pull-kubernetes-e2e-kind
   - pull-kubernetes-e2e-kind-ipv6
   - pull-kubernetes-integration
-  - pull-kubernetes-kubemark-e2e-gce-big
```
Comment (wojtek-t) [inline gubernator/config.yaml:37]: Given the job is not being removed, just no longer required, shouldn't we keep this as is?

Comment (spiffxp) [reply gubernator/config.yaml:37]: The recommended way to navigate to your job's logs would be to use spyglass.  Gubernator's list is generated based on `always_run` jobs.  We have basically stopped supporting gubernator at this point.

**Pull request merged**